response/core/urls.py

```diff
@@ -1,16 +1,22 @@
 from django.conf.urls import url, include
 from rest_framework import routers
 
-from response.core.views import IncidentViewSet, ActionViewSet, ExternalUserViewSet
+from response.core.views import (
+    IncidentViewSet,
+    IncidentActionViewSet,
+    ActionViewSet,
+    ExternalUserViewSet,
+)
 
 # Routers provide an easy way of automatically determining the URL conf.
 router = routers.DefaultRouter()
-router.register(r'incidents', IncidentViewSet, basename='incident')
-router.register(r'actions', ActionViewSet)
-router.register(r'ExternalUser', ExternalUserViewSet)
+router.register(r"incidents", IncidentViewSet, basename="incident")
+router.register(
+    r"incidents/(?P<incident_pk>[0-9]+)/actions", IncidentActionViewSet, basename="incident-action"
```

Comment:
Not required, but we could use the newer int syntax for this: https://docs.djangoproject.com/en/2.2/topics/http/urls/#example

Reply:
Unfortunately the Django REST Framework router doesn't appear to support this 😢 I tried it locally and it wouldn't register the URL - I suspect because it has it's own transformations it needs to do for generic viewsets (e.g. add `/{pk}` on the end for detail views)

https://www.django-rest-framework.org/api-guide/routers/